src/writeData/subscriptions/components/SubscriptionsLanding.tsx

```diff
@@ -51,15 +51,30 @@ const SubscriptionsLanding: FC = () => {
     sortType: SortTypes.String,
     sortDirection: Sort.Ascending,
   })
+  const [filteredSubscriptions, setFilteredSubscriptions] = useState([])
+
   useEffect(() => {
     event('visited subscriptions page', {}, {feature: 'subscriptions'})
   }, [])
-  const filteredSubscriptions =
-    subscriptions && search
-      ? subscriptions.filter(
-          s => s.name.toLowerCase() === search.toLowerCase().trim()
-        )
-      : subscriptions
+
+  useEffect(() => {
+    if (subscriptions === null || !subscriptions.length) {
```

Comment:
suggesting this change to handle `undefined` scenario as well as `null`.

```suggestion
    if (!subscriptions || !subscriptions.length) {
```